build.assets/Dockerfile-multiarch

```diff
@@ -130,6 +130,8 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --pr
     cargo --version && \
     rustc --version && \
     rustup target add ${RUST_ARCH}
+# Install wasm-pack for targeting WebAssembly from Rust.
+RUN cargo install wasm-pack
```

Comment:
@ibeckermayer Do you know which version will be installed? We're using this image to build our ARM64 images, but I remember I had some issues in the past where `cargo` ignored the OS arch and installed x86 anyway.
Can you run a dev build on this branch and confirm that this won't break our releases?

Reply:
I'll test it out. This should just use the latest version of wasm-pack for whatever architecture we're on.

Just to confirm I'm understanding correctly: by run a dev build you mean try something like `make -C build.assets ui` (which uses wasm-pack) on my local machine (which is ARM64) to confirm that it works?

Reply:
I was talking about a test dev build/release. I triggered one that includes code in this branch https://drone.platform.teleport.sh/gravitational/teleport/25432

Reply:
Did that work? Sorry, not up to speed on using Drone. I see that `build-linux-arm64` succeeded but I'm not quite sure what actually happened.

When I tried building the ui locally with this code via the docker image, the image build was failing for me with

```
Installing wasm-bindgen... Error: no prebuilt wasm-opt binaries are available for this platform: Unrecognized target!
```

which appears to be [a common arm 64 problem](https://github.com/rustwasm/wasm-pack/issues/913) and which I fixed by adding feb19a89b857bcfaa32480119e510e13f763c578.